src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -3,24 +3,24 @@
 import com.microsoft.azure.servicebus.IMessage;
 import com.microsoft.azure.servicebus.IMessageReceiver;
 import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.json.JSONException;
+import org.json.JSONObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.scheduling.annotation.Scheduled;
-import org.springframework.stereotype.Component;
-import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.EnvelopeParser;
 import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.exceptions.ConnectionException;
-import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
 
 import java.time.Duration;
 import java.time.Instant;
 import java.util.function.Supplier;
 
+import static java.nio.charset.Charset.defaultCharset;
+
 /**
  * Deletes messages from envelopes Dead letter queue.
  */
-@Component
```

Comment:
So it is not needed anymore? 🤔

Reply:
Good spot - without the annotation the schedule won't work.

Reply:
yeah, good spot! I didn't notice that I removed this annotation 😳